keepalive.go

```diff
@@ -8,24 +8,27 @@ import (
 	"golang.org/x/crypto/ssh"
 )
 
-// StartKeepalive starts sending server keepalive messages until done channel
-// is closed.
-func StartKeepalive(client *ssh.Client, interval time.Duration, countMax int, done <-chan struct{}) {
+// KeepAlive keeps an ssh channel alive sending keepalive pings
+// every interval seconds. The channel outlives up to maxErrors consecutive failures.
+// Done channel is a control channel that shutdowns a sidecar goroutine.
+func KeepAlive(client *ssh.Client, interval time.Duration, maxErrors int, done <-chan struct{}) {
```

Comment:
countMax is not some random string it relates to `ServerAliveCountMax` ssh option.

Reply:
You right. What do you think if we rename `ServerAliveCountMax` to `ServerAliveMaxErrors`? It may break compatibility, but just looking onto the name itself it seems to be it does not deliver its meaning

Reply:
Sure send PR to OpenSSH if they merge it I'm fine with that